.github/workflows/build.yml

```diff
@@ -189,7 +192,11 @@ jobs:
 
       - name: Import config
         run: |
-          ./scripts/bmc.sh ./index.cjs import --config ${CONFIG_LOCATION_PROD} --commit
+          ./scripts/update-lambda-config.sh
```

Comment:
This looks nice and easy for update the environment variables. Just wondering if we want to wrap this into a cli? 
And also we might still want to compare and comment changes for between old config and new config in the PR for preview changes.

Reply:
It was duplicated in two places is why I made it a bash shell, we could write a full cli for it, but figured it wasn't really worth it to wrap 2 aws sdk calls.

Reply:
the comparison between the config diffs, is also already handled futher up the script this is just the deployment phase

Reply:
I think we just need to update the `import` cli to `compare` cli, which is download production config from v1/version and do diffs between Pull request config. So we still got the diffs.